index.js

```diff
@@ -13,6 +13,7 @@ const path = require('path')
 const Peer = require('simple-peer')
 const randombytes = require('randombytes')
 const speedometer = require('speedometer')
+const queueMicrotask = require('queue-microtask');
```

Comment:
tests are failing because of this semicolon :stuck_out_tongue_winking_eye:

Reply:
argh